api_docs/source/javascripts/app/_search.js

```diff
@@ -1,102 +0,0 @@
-//= require ../lib/_lunr
```

Comment:
This functionality is unused - all the searching can be performed through the browser (`CTRL + F`) as everything is on a single page: https://procurify.github.io/sticky/#netsuite.